test/test_instrument/test_instrumentTest.php

```diff
@@ -165,9 +165,11 @@ function testTextElement()
     function testCheckBoxElement()
     {
         $this->_landing();
-        $this->safeFindElement(
+        $checkbox = $this->safeFindElement(
             WebDriverBy::Name("testCheckbox")
-        )->click();
+        );
+        $checkbox->click();
+        $this->assertTrue($checkbox->isSelected());
```

Comment:
try to run "        $checkbox = $this->safeFindElement(
            WebDriverBy::Name("testCheckbox")
        );" again

Reply:
Please remove line 172 to 175 to pass the test.

Reply:
Will do, but not sure it will pass this test. Lets see.